backend/apps/monkvision/lib/app.js

```diff
@@ -2,4 +2,8 @@
  * (C) 2020 TekMonks. All rights reserved.
  */
 
-module.exports.initSync = _ => { global.APP_CONSTANTS = require(`${__dirname}/../apis/lib/constants.js`); require(`${APP_CONSTANTS.LIB_DIR}/db.js`).init(); }
\ No newline at end of file
+module.exports.initSync = _ => {
+    global.APP_CONSTANTS = require(`${__dirname}/../apis/lib/constants.js`);
+    require(`${APP_CONSTANTS.LIB_DIR}/db.js`).init();
+    require(`${APP_CONSTANTS.LIB_DIR}/nlp.js`).init();
```

Comment:
initSync means methods are synchronous. Your init is not! This is not going to work.